build.gradle

```diff
@@ -17,7 +17,7 @@ repositories {
 dependencies {
     testImplementation 'junit:junit:4.12'
 
-    implementation 'com.vonage:client:5.5.0'
+    implementation 'com.vonage:client:5.6.0'
```

Comment:
If this is gong to be moving up to 6.0.0, this will need to change to reflect it.